rules/ec2CreatedByConsole.js

```diff
@@ -0,0 +1,37 @@
+var message = require('lambda-cfn').message;
+
+module.exports.config = {
+  name: 'ec2CreatedByConsole',
+  sourcePath: 'rules/ec2CreatedByConsole.js',
+  eventRule: {
+    eventPattern: {
+      'detail-type': [
+        'AWS Console Sign In via CloudTrail'
+      ],
+      detail: {
+        eventSource: [
+          'AWS API Call via CloudTrail'
+        ],
+        eventName: [
+          'RunInstances'
+        ]
+      }
+    }
+  }
+};
+
+module.exports.fn = function(event, callback) {
+  var notification = {
+    subject: 'User created instance EC2 by console.',
```

Comment:
@piligab can you determine who the user was, and put the actual username in the message?  I think this is at `event.userIdentity.userName`.  Then, the message should be `%username created an EC2 instance from the console` (just a small grammatical correction).